vitrivr-engine-module-fes/src/main/kotlin/org/vitrivr/engine/base/features/external/api/AsrApi.kt

```diff
@@ -35,6 +36,22 @@ class AsrApi(host: String, model: String, timeoutMs: Long, pollingIntervalMs: Lo
         }
     }
 
+    /**
+     * This method is used to start a batched ASR job on the API.
+     *
+     * @param input The input for the job.
+     * @return The [JobStatus]
+     */
+    override suspend fun startBatchedJob(input: List<AudioContent>): JobStatus {
+        logger.debug { "Starting batched ASR job for audio." }
+        val wrapped = BatchedAutomatedSpeechRecognitionInput(input.map { it.toDataURL() })
+        return try {
+            this.automatedSpeechRecognitionApi.newBatchedJobApiTasksAutomatedSpeechRecognitionBatchedModelJobsPost(this.model, wrapped).body()
+        } catch (e: Throwable) {
+            JobStatus("unknown", JobState.failed)
```

Comment:
Why "unknown"? The `Throwable` might know why...

Reply:
It's not the status that is unknown, it is the job id that is unknown if there is a failure during job creation.